go-controller/pkg/node/node.go

```diff
@@ -495,6 +498,15 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 	}
 
 	if config.HybridOverlay.Enabled {
+		var libovsdbNBClient libovsdbclient.Client
+		var err error
+
+		stopChan := make(chan struct{})
+		defer close(stopChan)
+
+		if libovsdbNBClient, err = libovsdb.NewNBClient(stopChan); err != nil {
```

Comment:
Is this the only client in this process or are you creating a new one on purpose?

Reply:
This is the only client in the Node process Code